seq2science/rules/configuration_explain.smk

```diff
@@ -145,7 +145,7 @@ else:
             end=" and finally were tn5 bias shifted by seq2science." if config.get("tn5_shift", 0) > 0 else ".",
         )
         + f"Afterwards samples were downsampled to {config['subsample']} reads."
-        if config.get("subsample")
+        if config.get("subsample", -1) != -1
```

Comment:
So strange that the explain tests don't catch this.. Why is that?

Reply:
it was working fine and printing "Afterwards samples were downsampled to -1 reads."

Reply:
ah never mind. it's explain. i thought it was in the rule itself :)